include/uapi/sound/sof/tokens.h

```diff
@@ -53,7 +53,6 @@
 #define SOF_TKN_SCHED_TIME_DOMAIN		205
 #define SOF_TKN_SCHED_DYNAMIC_PIPELINE		206
 #define SOF_TKN_SCHED_LP_MODE			207
-#define SOF_TKN_SCHED_MEM_USAGE			208
```

Comment:
but we do know at some point we'll need to track memory usage. This wasn't added because of random ideas but actual uses of IPC4. The fact that it's not used currently doesn't remove the fact that we'll need to have information on MCPS and memory to make more educated decisions or introduce resource management.

Reply:
no doubt about that but this token doesnt exist in the topology definitions today. We can add this back when we add it in topology to match what is used there